catalog/core/catalog-core-eventcommands/src/main/java/ddf/catalog/pubsub/command/ListCommand.java

```diff
@@ -69,10 +74,18 @@ protected Object doExecute() throws Exception {
             console.println();
             console.print(CYAN_CONSOLE_COLOR);
             console.print(SUBSCRIPTION_ID_COLUMN_HEADER);
+            console.print("\t\t| ");
+            console.print(CALLBACK_COLUMN_HEADER);
             console.println(DEFAULT_CONSOLE_COLOR);
 
-            for (String subscriptionId : subscriptionIds) {
-                console.println(subscriptionId);
+            for (String subscriptionId : subscriptionIds.keySet()) {
+                ServiceReference<Subscription> sr = subscriptionIds.get(subscriptionId);
+                Subscription subscription = bundleContext.getService(sr);
+                if(subscription!=null && !subscription.isActive())
+                {
+                    console.print(RED_CONSOLE_COLOR);
+                }
+                console.println(subscriptionId+"\t "+sr.getProperty("event-endpoint")+DEFAULT_CONSOLE_COLOR);
```

Comment:
Instead of concatenation, use a `String.format`. That way, you can also use a variable to hold the line color and make it the first replacement param in the line.